content/en/blog/sign-verify-plugins-rules/index.md

```diff
@@ -0,0 +1,178 @@
+
+Software supply chain attacks aim at injecting malicious code into software components like the above, to compromise downstream users. These types of attacks are among the primary threats in today's threat landscape.
+
+In particular, attackers abuse trust relationships existing between the different open-source stakeholders. The increase in attacks on open-source software throughout the last few years demonstrates that attackers consider them a viable means for spreading malware.
+
+## SSC safeguards
+
+Safeguards against supply chain attacks can be classified by control type: directive, preventive, detective, corrective, and recovery. But there ain't no such thing as a free lunch. Besides safeguards classification, the utility-to-cost ratio can also be an important factor in deciding where to start in improving the supply chain security of software, and can be pretty easy to measure it.
+
+There are cheap preventive safeguards that can be implemented in open source projects especially, where stakeholders platea can be pretty wide considering the contributions. For example, branch protection rules are usually simple per-code repository configurations in providers (e.g. GitHub) and alongside pull request-based flows enforcing code review quorum, are also standard best practices nowadays. The same applies to reproducible builds, dependency pinning, build steps isolation, MFA authentication to repository providers, etc.
+
+On the other side, safeguards with a high utility value can require a considerable amount of effort to be implemented end-to-end (producer and consumer sides).
+
+MITRE and OpenSSF increased their efforts to improve SSC security recently in general and in the open source, by providing frameworks and tools that increase the utility-to-cost ratio.
+MITRE for example proposes an end-to-end framework to preserve the integrity of the software supply chain. OpenSSF develops the SLSA framework that groups several security best practices for open-source projects.
+
+Moreover, the in-toto Attestation Framework defines a fixed, lightweight Statement that communicates some information about the execution of the software supply chain, such as if the source was reviewed, or if it went through a SLSA conformant build process.
+
+Supply chain security is important to be constantly increased through safeguards placed as soon as possible in the supply chain ("shift left") and needs to be maintained in a continuous manner reducing the operational burden (DevSecOps) because technologies evolve so attacks do.
+ 
+Software artifact signatures are one of the safeguards with a reasonable amount of effort nowadays, thanks also to important projects like Sigstore's [cosign](https://github.com/sigstore/cosign) for artifacts distributed as container images. So this was one of the goals decided for the Falco Software Supply Chain Working Group.
```

Comment:
"with a reasonable amount of effort' -> with "with good adoption" or "with good momentum"

I think you can drop "for artifacts distributed as container images"

"thanks to important projects like Sigstore's cosign (link), software artifact signatures are one of the SSC safeguards with a good amount of traction. Due to these factors, the Falco Software Supply Chain Working group decided to make implementing these signatures a goal for the Falco project"

Reply:
Thanks @mikegcoleman! The only thing I'd change is that I'd keep the utility-to-cost ratio as a metric for this type of safeguard instead of the traction or the adoption level of community projects that ease the process.

```suggestion
Thanks to important projects like Sigstore's [cosign](https://github.com/sigstore/cosign), software artifact signatures are one of the SSC safeguards with a high utility-to-cost ratio. Due to these factors, the Falco Software Supply Chain Working group decided to make implementing these signatures a goal for the Falco project"
```